pkg/kudoctl/cmd/plan/plan_status.go

```diff
@@ -63,15 +64,18 @@ func status(kc *kudo.Client, options *Options, ns string) error {
 	rootDisplay := fmt.Sprintf("%s (Operator-Version: \"%s\" Active-Plan: \"%s\")", instance.Name, instance.Spec.OperatorVersion.Name, lastPlanStatus.Name)
 	rootBranchName := tree.AddBranch(rootDisplay)
 
-	for name, plan := range ov.Spec.Plans {
-		if name == lastPlanStatus.Name {
-			var planDisplay string
+	plans := make([]string, 0, len(ov.Spec.Plans))
+	for k := range ov.Spec.Plans {
+		plans = append(plans, k)
+	}
+	sort.Strings(plans)
+
+	for _, plan := range plans {
+		if plan == lastPlanStatus.Name {
+			planDisplay := fmt.Sprintf("Plan %s (%s strategy) [%s]%s", plan, ov.Spec.Plans[plan].Strategy, lastPlanStatus.Status, printMessageIfAvailable(lastPlanStatus.Message))
 			if lastPlanStatus.LastUpdatedTimestamp != nil {
-				planDisplay = fmt.Sprintf("Plan %s (%s strategy) [%s]%s, last updated %s", name, plan.Strategy, lastPlanStatus.Status, printMessageIfAvailable(lastPlanStatus.Message), lastPlanStatus.LastUpdatedTimestamp.Format("2006-01-02 15:04:05"))
-			} else {
-				planDisplay = fmt.Sprintf("Plan %s (%s strategy) [%s]%s", name, plan.Strategy, lastPlanStatus.Status, printMessageIfAvailable(lastPlanStatus.Message))
+				planDisplay = planDisplay + fmt.Sprintf(", last updated %s", lastPlanStatus.LastUpdatedTimestamp.Format("2006-01-02 15:04:05"))
```

Comment:
```suggestion
				planDisplay = fmt.Sprintf("%s, last updated %s", planDisplay, lastPlanStatus.LastUpdatedTimestamp.Format("2006-01-02 15:04:05"))
```

Reply:
Not sure which one is actually more readable...

Reply:
I'm also okay with the little bit of code duplication that was there before. Regarding readability I'd suggest
```suggestion
				planDisplay += fmt.Sprintf(", last updated %s", lastPlanStatus.LastUpdatedTimestamp.Format("2006-01-02 15:04:05"))
```
or what is there right now, because it's clearer thatn `fmt.Sprintf` that `planDisplay` is mutated.

Reply:
it is really strange to `+` concat strings when a `sprintf` is on the RH side...  I'm with @ANeumann82 and his recommendation

Reply:
I've gone with @ANeumann82's recommendation 👍